ReSwift/CoreTypes/Store.swift

```diff
@@ -28,7 +28,9 @@ public class Store<State: StateType>: StoreType {
     public var dispatchFunction: DispatchFunction!
 
     private var reducer: AnyReducer
+  
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)